### PR TITLE
Support for SDK v916

### PR DESCRIPTION
**Pull request by manoj897**

### CODE REVIEW

File: samples/bannerSample/build.gradle [hunk 1]
```diff
@@ -38,7 +38,7 @@ dependencies {
     implementation 'androidx.appcompat:appcompat:1.0.0'
     implementation 'androidx.annotation:annotation:1.0.0'
     implementation 'androidx.legacy:legacy-support-v4:1.0.0'
-    implementation 'com.inmobi.monetization:inmobi-ads:9.1.1'
+    implementation 'com.inmobi.monetization:inmobi-ads:9.1.6'
 
     implementation 'com.google.android.gms:play-services-ads-identifier:17.0.0'
     implementation 'com.google.android.gms:play-services-location:16.0.0'
```
Comment (RajSuvariya-InMobi) [inline samples/bannerSample/build.gradle]: Let's use project-level Gradle to maintain dependency versions, also match the versions with unified-sdk deps versions

**Review by RajSuvariya-InMobi — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 2]
```diff
@@ -23,6 +23,15 @@ ext {
 
     gmsVersion = "12.0.1"
     androidSupportLibVersion = "29.0.0"
+
+    inmobiSdkVersion = "9.1.6"
+    fresco = "0.12.0"
+    picasso = "2.71828"
+    browser = "1.3.0"
+    recyclerview = "1.0.0"
```
Comment (RajSuvariya-InMobi) [inline build.gradle]: We have "1.1.0"  in unified-sdk

**Review by RajSuvariya-InMobi — CHANGES_REQUESTED** [review]

File: build.gradle [hunk 3]
```diff
@@ -14,7 +14,7 @@ buildscript {
 
 
 ext {
-    androidMinSdkVersion = 15
+    androidMinSdkVersion = 16
```
Comment (im-lakshya) [inline build.gradle]: SDK minimum support is 15, but it is 16 ?

**Comment by im-lakshya** [comment]
InMobiSDK.aar is deleted, but we replace it with new version everytime

**Review by RajSuvariya-InMobi — APPROVED** [review]

**Comment by manoj897** [comment]
> InMobiSDK.aar is deleted, but we replace it with new version everytime

I can see that aar file is checked in

**Comment by im-lakshya** [comment]
> I can see that aar file is checked in

My bad, I've reviewed it commit wise and in there it was showing it as deleted but when reviewed accumulated file changes it is showing as updated.

**Review by im-lakshya — APPROVED** [review]

**Pull request merged**